src/translations/libfm-qt_fr.ts

```diff
@@ -37,12 +37,12 @@
 &lt;li&gt;&lt;b&gt;%u&lt;/b&gt;: Represents a single URI of the file&lt;/li&gt;
 &lt;li&gt;&lt;b&gt;%U&lt;/b&gt;: Represents multiple URIs&lt;/li&gt;
 &lt;/ul&gt;</source>
-        <translation>&lt;b&gt;Ces codes spéciaux peuvent être utilisés dans la ligne de commande&#x202f;:&lt;/b&gt;
+        <translation>&lt;b&gt;Ces codes spéciaux peuvent être utilisés dans la ligne de commande :&lt;/b&gt;
 &lt;ul&gt;
-&lt;li&gt;&lt;b&gt;%f&lt;/b&gt;&#x202f;: représente un seul nom de fichier,&lt;/li&gt;
```

Comment:
@tsujan What do you think about that? He removed the non breakable space with a normal space. At the end I think it doesn't matter as those lines are hardly broken, I've no idea if there are always non breakable spaces in french translation... Weblate accepts both types of spaces.

Reply:
I think it's correct in French. Have seen it in many places outside LXQt too.

Reply:
I meant replace  `&#x202f; `with a normal space which could end up like
```
Ces codes spéciaux peuvent être utilisés dans la ligne de commande
: représente un seul nom de fichier
```

https://www.fileformat.info/info/unicode/char/202f/index.htm

Reply:
No being an expert at French at all, I can only say that the Internet showed me contradicting opinions. You're right, a non-breaking space should be preferable but, for example, see https://wiki.openoffice.org/wiki/Non_Breaking_Spaces_Before_Punctuation_In_French_(espaces_insécables). It recommends `U+00A0`, not `202F` ;)

Reply:
I searched all LXQt components, only libfm-qt_fr.ts has some of those `&#x202f;` so I'll merge that.